workflows/iuf/operations/management-nodes-rollout/management-m001-rollout.yaml

```diff
@@ -81,7 +81,15 @@ spec:
                   value: "{{=jsonpath(inputs.parameters.global_params, '$.input_params.media_host')}}"
                 - name: scriptContent
                   value: |
+                    . /etc/cray/upgrade/csm/myenv
                     echo "NOTICE This workflow will rollout ncn-m001 upgrade."
+                    if [ -f /etc/cray/upgrade/csm/${CSM_REL_NAME}/ncn-m001-backup.done ] || [ -f /etc/cray/upgrade/csm/${CSM_REL_NAME}/ncn-m001-bootparameters-update.done ]; then
```

Comment:
I don't think of any of this output is needed here. I think it would confuse the user more than help them. I would put these comments after the note in the next stage that says it is being skipped. So put " Note: To Re-run master node ncn-m001 bootparameters update, run the command: 'rm -f /etc/cray/upgrade/csm/${CSM_REL_NAME}/ncn-m001-bootparameters-update.done'" after "INFO Skipping master node ncn-m001".